Add Starter kit, Glossary & Kickstart your community page (#1189)
galaxyproject/galaxy-hub: content/get-started/new-leads/index.md

---
title: Set up your own community
---

> There are many communities stablished within Galaxy based on [**region**](/community/#regional-communities) or [**domain of science **](/community/#regional-communities) but if you feel like you just don't see your own or don't feel identified with them you can create your own community!

Reviewer: established instead of stablished.

> There are many communities established within Galaxy based on [**region**](/community/#regional-communities) or [**domain of science**](/community/#regional-communities) but if you feel like you just don't see your own or don't feel identified with them, you can create your own community!

Reviewer: The link to domain specific communities is not correct (for now its point to regional communities).

Below you'll find a suggested path to follow before you can start your own community, a list of Do's and Dont's and also useful links you should probably check out.

### Path 

#### Learn the basics about Galaxy

> First of all, you should  start with our dedicated [Get Started](/get-started/) page to get a head-start of what Galaxy is about. You'll find all the necessary resources and you could even participate in the [Galaxy Mentorship Network](#) to get started with the help of a Galaxy Mentor.

Reviewer 2: mentoring

#### Learn about Tool Development

> Galaxy as many tools already installed and ready to be used but when creating your community you'll probably need specific tools for your Galaxy analysis. Don't worry, see our dedicated page for [Tool Authors](/tools/) where you'll find everything you'll need and also reach out to the [Tool Working Group](/community/wg/) via Gitter.

Author: I'd say yes if we change that from other main places were they're mentioned as well. Working groups page says "Gitter Channels" for e.g. .-

Also the URLs should be change.  Either to https://matrix.to/#/#galaxyproject_wg-systems:gitter.im?utm_source=gitter or https://app.element.io/#/room/#galaxyproject_wg-systems:gitter.im if we want to suggest using element?

> Galaxy as many tools already installed and ready to be used but when creating your community you'll probably need specific tools for your Galaxy analysis. Don't worry, see our dedicated page for [Tool Authors](/tools/) where you'll find everything you'll need and also reach out to the [Tool Working Group](/community/wg/) via Gitter.

Reviewer 2: I'm not sure but I think it's tools wg (plural)

Author: I've checked, it is [here](https://github.com/orgs/galaxyproject/projects/13)

Don't forget to follow the [Best Practices](https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices.html) for writing Galaxy Tools.

#### Get involved
- Introduce yourself and dive in our Working Groups.
- Participate in our World Wide Galaxy Events
- We strongly recommend you create your community chat. Reach out to our Lobby in [Gitter](https://gitter.im/galaxyproject/Lobby) and ask the admins to create your Community Chat.
  - Setting up a mailing list is also another available option by sending an email to outreach@galaxyproject.org altough we strongly recommend to use a Community Chat.

### Do's and Dont's 

This curated table was made with the help of Community member's experience in hope to avoid obstacles in your path to create your Community.

| __DO__  | __DON'T__   |
|---|---|
|Be patient! Take the time to learn the basics about Galaxy. | Hurry and dive in straight  to create without knowing a lot about Galaxy and it's workarounds. |

> | Ask if a tool for what you need already exists| Get hazy with all your tool's possible needs as there are many installed already that can be close to what you would like.|

Reviewer 2: ```suggestion
| Ask if a tool for what you need already exists. | Get hazy with all your tool's possible needs as there are many installed already that can be close to what you would like.|
```

> |Request your tool to be reviewed, avoid managint your repo alone!| Be a lone wolf. Oherwise the main servers may not want to install your tool, as they want to make sure it follows standars they have defined for developing tools. |

Reviewer: managint --> managing

standars --> standards

> |Request your tool to be reviewed, avoid managint your repo alone!| Be a lone wolf. Oherwise the main servers may not want to install your tool, as they want to make sure it follows standars they have defined for developing tools. |

Reviewer 2: ```suggestion
|Request your tool to be reviewed, avoid managing your repo alone!| Be a lone wolf. Oherwise the main servers may not want to install your tool, as they want to make sure it follows standars they have defined for developing tools. |
```

> |Request your tool to be reviewed, avoid managing your repo alone!| Be a lone wolf. Oherwise the main servers may not want to install your tool, as they want to make sure it follows standars they have defined for developing tools. |

Reviewer 2: otherwise indicates here the opposite I think

> |When you happen to need a tool make the tool > write the training > set up an event.  Repeat events and take advantage of Galaxy’s infrastructure (Smorgasbord,CoFests,GCC,etc) to improve your tool and receive feedback from the Community.|Set to create very specific tools and avoid feedback, try not to overwhelm yourself!|

Reviewer: space missing after comma:

(Smorgasbord,CoFests,GCC,etc) --> (Smorgasbord, CoFests, GCC, etc)

Reviewer: I am not sure I fully understand the meaning of:

, try not to overwhelm yourself!

in the context of DON'T

> |When you happen to need a tool make the tool > write the training > set up an event.  Repeat events and take advantage of Galaxy’s infrastructure (Smorgasbord,CoFests,GCC,etc) to improve your tool and receive feedback from the Community.|Set to create very specific tools and avoid feedback, try not to overwhelm yourself!|

Reviewer 2: ```suggestion
|When you happen to need a tool: Make the tool -> write the training -> set up an event. Repeat events and take advantage of Galaxy’s infrastructure (Smorgasbord, CoFests, GCC, etc.) to improve your tool and receive feedback from the Community. |Set to create very specific tools and avoid feedback.
```

> |Use already established servers. Asking for a subdomain within the usegalaxy.eu server is a great option!|Set up a server all by yourself (consider resources, complexity,security issues and more) | Think about maximizing the use of your tool.|Create specific tools for very specific jobs |

Reviewer: space missing after comma (before security) e.g.

(consider resources, complexity, security issues and more)

> |Use already established servers. Asking for a subdomain within the usegalaxy.eu server is a great option!|Set up a server all by yourself (consider resources, complexity, security issues and more) | Think about maximizing the use of your tool.|Create specific tools for very specific jobs |

Reviewer 2: this last row is still strange

> |Use already stablished servers. Ask for a subdomain within the usegalaxy.eu server is a great option!|Set up a server all by yourself (consider resources, complexity,security issues and more) |||Think about maximizing the use of your tool.|Create specific tools for very specific jobs |

Reviewer 2: ```suggestion
|Use already established servers. Asking for a subdomain within the usegalaxy.eu server is a great option!|Set up a server all by yourself (consider resources, complexity,security issues and more) | Think about maximizing the use of your tool.|Create specific tools for very specific jobs |
```

Reviewer 2: there is something strange here :)
